gems/pending/VolumeManager/MiqLvm.rb

```diff
@@ -211,12 +211,17 @@ def getPvObj(vgObj, pvName, pv)
     pvObj = PhysicalVolume.new(pv['id'].delete('-'), pvName, pv['device'], pv['dev_size'], pv['pe_start'], pv['pe_count'])
     # Add reference to volume group object to each physical volume object.
     pvObj.vgObj = vgObj
-    # Add reference to the physical volume's open disk object to the physical volume object.
-    dobj = @pvHdrs[pvObj.pvId].diskObj
-    # Add reference to the physical volume object to the physical volume's open disk object.
-    pvObj.diskObj = dobj
+
+    if @pvHdrs.keys.include?(pvObj.pvId)
+      # Add reference to the physical volume's open disk object to the physical volume object.
+      dobj = @pvHdrs[pvObj.pvId].diskObj
+
+      # Add reference to the physical volume object to the physical volume's open disk object.
+      pvObj.diskObj = dobj
+      dobj.pvObj = pvObj
```

Comment:
The two comments don't seem to line up with the code they are referencing, but maybe I'm misunderstanding...I would expect the comment at line 216 to be commenting on 220 and the one at line 219 to be commenting on the line at 221.  Are the comments even necessary?